extensions/adobe/experience/decisioning/proposition-interaction-detail.example.2.json

```diff
@@ -44,6 +44,10 @@
         "xdm:algorithmID": "auto-optimize",
         "xdm:trafficType": "noncontextual"
       }
-    ]
+    ],
+    "xdm:characteristics": {
+      "xdm:prop1": "Value 1",
```

Comment:
Same feedback as above, let's drop the `xdm:` prefix to indicate it doesn't need any specific xdm fields and is a simple container for any data.

Reply:
Sharing an example for option (item) characteristics here for your reference. https://github.com/adobe/xdm/blob/master/docs/reference/adobe/experience/decisioning/option-detail.schema.md#decision-option-details-example

Reply:
ok, thank you